### PR TITLE
Add issue templates (bug, feature request, general)

### PR DESCRIPTION
**Pull request by vince-fugnitto**
Added three issue templates for the repository:
1. `bug report`: create an issue for a bug report
2. `feature request`: create an issue for a feature request
3. `general`: create a general issue (Ex: questions)

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
@lmcbout sorry for always pinging you for reviews 🙈

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+**How to reproduce:**
+<!-- Steps to reproduce the behavior: -->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Environment**
+ - _OS_: 
+ - _Browser_:
+
```
Comment (lmcbout) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would also add: "-Theia version"
When there is a Bug, nice to know on which version they report the issue

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+**Additional information**
+<!--Add any other context about the problem here.-->
```
Comment (lmcbout) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Additional info: We may suggest to add a trace or a Debug file if available

**Comment by vince-fugnitto** [comment]
@lmcbout I updated the templates based on your comments.

**Pull request merged**